### ISSUE TITLE
Add language: Cosmo

### ISSUE THREAD

**Issue by R-unic** [github/linguist#6429]
## Language name
[Cosmo](https://github.com/cosmo-lang/cosmo)

## URL of example repository
https://github.com/cosmo-lang/cosmo/tree/master/examples

## URL of syntax highlighting grammar
https://github.com/cosmo-lang/vscode-cosmo/blob/main/syntaxes/cosmo.tmLanguage.json

## Most popular extensions
`.cos`, `.⭐`

## Detected language
None

**Comment by Nixinova** [comment]
Dot-star file extension is a bit of a weird one and doesn't seem to be used outside of the cosmo-lang examples folder ([search link](https://github.com/search?q=path%3A%2F.%2B%5C.%E2%AD%90%24%2F&type=code)). .cos seems to have a lot of junk ([search link](https://github.com/search?q=path%3A%2F.%2B%5C.cos%24%2F&type=code)).

**Comment by R-unic** [comment]
yeah i noticed that shortly after posting this. so the only keywords i could imagine would be unique to cosmo in a `.cos` file might be `fn`, `from`, `until`, `unless`, `is`, `in`, or `mut`. you could also use the function name `puts`. i'm aware all of those keywords are present in other languages, so the only keywords universally unique to cosmo are `every` and `$`.

**Comment by johnmays** [comment]
I bet true `.cos` files are getting close to 2000.  Here is a search for [`path:*.cos fn`](https://github.com/search?q=path%3A*.cos+NOT+is%3Afork+fn&type=code) with ~1.4k.  And when you use other keywords like `stream`, you can tell there are `.cos` files floating around without the `fn keyword`.  As for `.⭐` there are [only 15 files](https://github.com/search?q=path%3A*.%E2%AD%90+NOT+is%3Afork&type=code) at the moment.  And according to #5756, each extension needs at least 2000.
